modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -63,150 +63,187 @@ Ordering is important when adding the kernel arguments: `ip=`, `nameserver=`, an
 
 The networking options are passed to the `dracut` tool during system boot. For more information about the networking options supported by `dracut`, see the `dracut.cmdline` manual page.
 
-.Networking and bonding options for ISO installations
 endif::ibm-z-kvm[]
 ifdef::ibm-z-kvm[]
-The following table provides examples for configuring networking on your {op-system} nodes for ISO installations. The examples describe how to use the `ip=` and `nameserver=` kernel arguments.
+The following information provides examples for configuring networking on your {op-system} nodes for ISO installations. The examples describe how to use the `ip=` and `nameserver=` kernel arguments.
 
 [NOTE]
 ====
 Ordering is important when adding the kernel arguments: `ip=` and `nameserver=`.
 ====
 
 The networking options are passed to the `dracut` tool during system boot. For more information about the networking options supported by `dracut`, see the `dracut.cmdline` manual page.
-
-.Networking options for ISO installations
 endif::ibm-z-kvm[]
-|===
-|Description |Examples
 
-a|To configure an IP address, either use DHCP (`ip=dhcp`) or set an individual static IP address (`ip=<host_ip>`). If setting a static IP, you must then identify the DNS server IP address (`nameserver=<dns_ip>`) on each node. This example sets: +
+.Networking and bonding options for ISO installations
```

Comment:
You might need to take another look at how the headings for all of this information is laid out. The way the levels should work is that they get decreasingly smaller as you go further down. But the way you have it now, a later heading is actually bigger than one of the earlier ones, plus having the red one thrown in - makes it hard to know how the content is actually grouped:

![image](https://user-images.githubusercontent.com/7443887/203361062-9f5f3e35-af66-460b-baac-27ea2a7c7624.png)

This .Title syntax (`.Networking and bonding options for ISO installations`) is not a heading that should have a bunch of other headings underneath it. It is a title that is used for a single thing, like a code block, table, or paragraph. So I don't think you want to be using this at all now since you removed the table.  (It was fine as a title when it was just a table).

I agree that this content wasn't great in a table, but I think that it's also a lot harder as you have it now to distinguish what section belongs with what.

Also because you're using the .Title syntax here, the next line:

```

[discrete]
==== Configuring an DHCP and static IP address
```
Is technically at the wrong level down. If you remove the "[discrete]" tag, and build it, you'll see that it produces a warning about being out of sequence:

```
asciidoctor: WARNING: modules/installation-user-infra-machines-static-network.adoc: line 82: section title out of sequence: expected level 5, got level 6
```

If you change `.Networking and bonding options for ISO installations` to be `=== Networking and bonding options for ISO installations`, then these titles would be in the proper order. HOWEVER, even when doing this, the "Configuring an DHCP and static IP address" heading is a level 7 heading, and ends up being a larger font. I believe this means that level 7 headings aren't supported.

Sorry for the long comment, but I think you might need to revisit how best to reorganize this content so that it's no so many levels deep.